sdk/python/sawtooth_sdk/processor/state.py

```diff
@@ -90,3 +90,31 @@ def set(self, entries, timeout=None):
             raise InvalidTransaction(
                 'Tried to set unauthorized address: {}'.format(addresses))
         return response.addresses
+
+    def delete(self, entries, timeout=None):
+        """
+        delete an address in the validator's merkle state
+        Args:
+            entries (list): list of StateEntry
+            timeout: optional timeout, in seconds
+
+        Returns:
+            addresses (list): a list of addresses that were deleted
+
+        """
+        state_entries = [state_context_pb2.Entry(
+            address=e.address,
+            data=e.data) for e in entries]
+        request = state_context_pb2.TpStateDeleteRequest(
+            entries=state_entries,
+            context_id=self._context_id).SerializeToString()
+        response = state_context_pb2.TpStateDeleteResponse()
+        response.ParseFromString(
+            self._stream.send(Message.TP_STATE_DELETE_REQUEST,
+                              request).result(timeout).content)
+        if response.status == \
+                state_context_pb2.TpStateDeleteResponse.AUTHORIZATION_ERROR:
+            addresses = [e.address for e in entries]
```

Comment:
What I am doing is the established pattern for state handlers. 
https://github.com/hyperledger/sawtooth-core/blob/master/sdk/python/sawtooth_sdk/processor/state.py#L60

https://github.com/hyperledger/sawtooth-core/blob/master/sdk/python/sawtooth_sdk/processor/state.py#L89